pkg/mount/mountinfo_linux.go

```diff
@@ -52,8 +56,17 @@ func parseInfoFile(r io.Reader, filter FilterFunc) ([]*Info, error) {
 		p.Major, _ = strconv.Atoi(mm[0])
 		p.Minor, _ = strconv.Atoi(mm[1])
 
-		p.Root = fields[3]
-		p.Mountpoint = fields[4]
+		r, err := strconv.Unquote(`"` + fields[3] + `"`)
+		if err != nil {
+			return nil, errors.Wrapf(err, "Parsing '%s' failed: unable to unquote root field", fields[3])
+		}
+		p.Root = r
+
+		mp, err := strconv.Unquote(`"` + fields[4] + `"`)
```

Comment:
There's no need to have a temp variable here, since you exit on error anyway, so maybe
```go
root.Mountpoint, err := strconv.Unquote(`"` + fields[4] + `"`)
if err != nil {
￼ 			return nil, errors.Wrapf(err, "Can't parse mount point %q", fields[4])
}
```

(I have also made error message smaller, but it's up to you)

Reply:
Yes, I wanted to do a follow up to fix the error messages (also make them lowercase); for this fix, I went for consistency with the existing ones.